app/src/main/res/values/strings.xml

```diff
@@ -41,4 +41,12 @@
     <string name="close">Close</string>
     <string name="general_User">User</string>
     <string name="general_currency" translatable="false">€</string>
+    <string name="sign_in_registerButton_Text">Registrate now</string>
```

Comment:
sign up für registrate und sign in für login. Damit wir das ganze einheitlich halten